google/cloud/forseti/services/model/service.py

```diff
@@ -103,8 +103,15 @@ def DeleteModel(self, request, _):
         """
 
         model_name = request.handle
-        self.modeller.delete_model(model_name)
-        return model_pb2.DeleteModelReply()
+        try:
+            self.modeller.delete_model(model_name)
+            success = model_pb2.DeleteModelReply.Status.Value('SUCCESS')
```

Comment:
nit: i would change this line:
```
status = model_pb2.DeleteModelReply.Status.Value('SUCCESS')
```
So that you can save two lines of code like this:
```
try:
    self.modeller.delete_model(model_name)
    status = model_pb2.DeleteModelReply.Status.Value('SUCCESS')
except Exception:
    LOGGER.exception('Unable to delete model: %s', model_name)
    status = model_pb2.DeleteModelReply.Status.Value('FAIL')

return model_pb2.DeleteModelReply(status=status)
```

Reply:
Done